### PR TITLE
Apply some general rebranding

### LINKED ISSUE #786: General rebranding
- fonts
- colors
- spacings
- icons

### PR DESCRIPTION
**Pull request by etharner**
Partially closes #786,

### CODE REVIEW

**Review by CedrikNikita — COMMENTED** [review]
Buttons in several screens are misaligned in this PR

**Comment by mradkov** [comment]
Screenshot 2021-02-03 at 8 33 10

**Review by CedrikNikita — COMMENTED** [review]
I also think that this PR may introduce new size of the wallet.

File: src/popup/popup.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
   <title>Superhero Wallet</title>
   <link rel="stylesheet" href="<%= PLATFORM === 'web' ? '/' : '' %>popup.css">
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no, viewport-fit=cover" />
+  <link href="https://fonts.googleapis.com/css?family=IBM+Plex+Sans:300,400,500,600,700&display=swap" rel="stylesheet">
```
Comment (imollov) [inline src/popup/popup.html:8]: Just wondering if we should bundle the font in the extension package for performance reasons, as well as centralization concerns?

Comment (mradkov) [reply src/popup/popup.html:8]: Yeah, please do not use googleapis.

Comment (mradkov) [reply src/popup/popup.html:8]: You can see how it is included in the superhero-ui [here](https://github.com/aeternity/superhero-ui/blob/fb2e17536b427dd6498fb15c6075071ef3ed6dfd/src/styles/_fonts.scss) and [here](https://github.com/aeternity/superhero-ui/blob/122adac107f3a92807ede809676a88763f929e81/package.json#L29) and use it in the same way.

File: src/popup/App.vue [hunk 2]
```diff
@@ -130,24 +130,39 @@ export default {
 </script>
 
 <style lang="scss">
-html {
-  height: 100%;
+html,
+body {
+  width: 360px;
```
Comment (imollov) [inline src/popup/App.vue:135]: Does it make sense to extract the fixed width into a variable? There are more than a few occurrences, so far.

File: src/popup/App.vue [hunk 3]
```diff
@@ -130,24 +130,39 @@ export default {
...
+
+@media screen and (min-width: 360px) {
+  html {
+    min-height: 100%;
+    margin: 0 auto;
+  }
+
+  body {
+    min-height: 100%;
+    height: 1px;
```
Comment (imollov) [inline src/popup/App.vue:152]: What is this about?

File: src/popup/App.vue [hunk 4]
```diff
@@ -130,24 +130,39 @@ export default {
...
+body {
+  width: 360px;
+  height: 600px;
 }
 
 body {
-  min-height: 100%;
-  height: 1px;
+  margin: 0;
+  background-color: black;
```
Comment (imollov) [inline src/popup/App.vue:141]:
```suggestion
  background-color: $color-black;
```

**Review by CedrikNikita — CHANGES_REQUESTED** [review]
Additional changes in the `width and height` commit
https://github.com/aeternity/superhero-wallet/blob/e2861d917f471a98b279a25f3021d5aad242f6af/src/popup/router/components/Header.vue#L105

**Review by CedrikNikita — APPROVED** [review]

**Review by imollov — APPROVED** [review]

**Review by mradkov — APPROVED** [review]

**Pull request merged**